index.html

```diff
@@ -432,7 +432,7 @@
-,{"system":"Hydra","date":"2023-09-19","machine":"c6a.4xlarge, 500gb gp2","cluster_size":1,"comment":"","tags":["C","PostgreSQL compatible","column-oriented"],"load_time":1222,"data_size":18995669982,"result":[[0.381752,0.249001,0.243783],[0.261088,0.211013,0.209277],[0.6692,0.634666,0.627473],[0.545165,0.431892,0.439854],[22.4371,21.8547,21.731],[34.8689,33.0467,32.5865],[0.410038,0.377206,0.376254],[0.22464,0.206133,0.204092],[27.1582,26.4292,26.5203],[30.1541,29.4784,30.2199],[3.31616,2.76618,2.75128],[3.55764,3.28978,3.27316],[7.74401,7.64012,7.6384],[10.703,10.2599,10.2098],[8.70712,8.33702,8.29268],[17.9937,17.5084,17.5291],[22.8406,22.2649,22.1176],[4.08958,3.59194,3.61345],[39.0156,37.7258,37.8926],[0.267125,0.233304,0.233594],[3.12914,2.74305,2.72563],[2.91522,2.89177,2.90416],[3.76881,3.62139,3.62348],[26.322,20.5805,20.3337],[1.69125,1.05602,1.06068],[1.06135,0.828339,0.831532],[1.26898,1.06035,1.05212],[4.17896,4.03193,4.03657],[43.0503,42.5091,42.3298],[7.89438,7.50181,7.39528],[9.33451,8.76728,8.79973],[15.0403,14.6597,14.3034],[58.8221,58.6056,58.6859],[34.2914,35.0405,34.2519],[36.3141,35.6205,36.5728],[14.2056,13.8882,13.8284],[0.367202,0.33666,0.33517],[0.22008,0.172653,0.167434],[0.15307,0.126572,0.126678],[1.11688,0.671288,0.696326],[0.160667,0.145181,0.141243],[0.137686,0.122734,0.118493],[0.197786,0.123078,0.122094]],"source":"hydra/results/c6a.4xlarge.json"}
+,{"system":"Hydra","date":"2025-03-04","machine":"cloud","cluster_size":"serverless","tags":["managed","PostgreSQL compatible","column-oriented"],"load_time":136,"data_size":30816390348,"result":[[0.192382,0.031411,0.0182],[0.277037,0.017238,0.016364],[0.245723,0.038173,0.037376],[0.306797,0.042102,0.042718],[0.418011,0.242568,0.23039],[0.551909,0.282084,0.278836],[0.180043,0.02662,0.025371],[0.165209,0.024473,0.023097],[0.453139,0.284364,0.283832],[0.535552,0.393579,0.39756],[0.30116,0.092113,0.091926],[0.257973,0.100089,0.098468],[0.398887,0.26349,0.253034],[0.578082,0.426437,0.425838],[0.434567,0.263478,0.266679],[0.379118,0.262621,0.264961],[0.729468,0.564334,0.562802],[0.602923,0.464127,0.465615],[1.33815,0.996799,0.993801],[0.173591,0.013425,0.013638],[1.68838,0.515917,0.516346],[0.437349,0.294343,0.296138],[1.99367,0.417726,0.417281],[8.64109,1.77034,1.77321],[0.264145,0.104584,0.105626],[0.238236,0.095826,0.095679],[0.287271,0.112243,0.108904],[0.755408,0.612582,0.61124],[6.75692,6.60809,6.60236],[0.23476,0.07472,0.067433],[0.382783,0.239876,0.239216],[0.438111,0.296216,0.288612],[1.5727,1.69792,1.40727],[1.55361,1.28201,1.28065],[1.49317,1.40405,1.38793],[0.579766,0.41326,0.413215],[0.226025,0.062452,0.059734],[0.195866,0.024387,0.041084],[0.198459,0.025349,0.026228],[0.323418,0.091749,0.088929],[0.172858,0.019531,0.019639],[0.172619,0.019008,0.022273],[0.194027,0.027197,0.024702]],"source":"hydra/results/hydra.json"}
```

Comment:
No need to update index.html by hand, it is automatically updated by a CI job based on *.json files in `results/` subfolders.

Reply:
got it, I'll undo this change.